monetio/models/_wrfchem_mm.py

```diff
@@ -88,6 +88,13 @@ def open_mfdataset(
         var_list.append("PSFC")
         # need to calculate surface pressure and dp and optionally dz here.
 
+        # Additional defaults for satellite analysis
+        var_list.append("PH")
+        var_list.append("PHB")
+        var_list.append("PB")
+        var_list.append("P")
```

Comment:
None of these are the same as "pres"?

Reply:
If this is setup for a WRF grid, then PH and PHB are the perturbation geopotential height and base geopotential height respectively (i. e., the height is (PH + PHB) / 9.8). P and PB are the perturbation and base pressures (pressure should be P + PB).

I haven't checked this reader in detail (nor will I be able to do so soon), but hopefully this helps.
Cheers,
Pablo